tests/Tests/500__jupyterhub/test-versions.robot

```diff
@@ -18,7 +18,7 @@ Force Tags          JupyterHub
 &{package_versions}      # robocop: disable
 ${JupyterLab_Version}         v3.2
 ${Notebook_Version}           v6.4
-${JupyterLab-git_Version}     v0.30
+${JupyterLab-git_Version}     v0.32
```

Comment:
In order to support RHODS 1.14 and RHODS 1.15 (used during live testing) you can define a variable with the old value (e.g. `${JupyterLab-git_Version-legacy})` and use it with` Is RHODS Version Greater Or Equal Than`